Sources/WalletSdk/Logger.swift

```diff
@@ -0,0 +1,98 @@
+// File: Logger
+//
+//    Copy log messages to the system logger.
+
+//
+// Imports
+//
+
+import Foundation
+
+//
+// Code
+//
+
+// Class: SpruceKitLogger
+//    A logging system for debug purposes.
```

Comment:
Please just use docstrings

Reply:
+1

Reply:
I have changed the documentation to swift style docstrings.
The logger isn’t doing anything other than console logging at the moment because this is the initial version of it, but the acceptance criteria include file logging.  The apple logger goes to console and syslog, but not files.